temporalcli/commandsmd/commands.md

```diff
@@ -655,7 +663,107 @@ The temporal task-queue list-partition command displays the partitions of a Task
 
 * `--task-queue`, `-t` (string) - Task queue name. Required.
 
-### temporal task-queue update-build-ids: Operations to update the sets of worker Build ID versions on the Task Queue.
+
+### temporal task-queue update-build-id-rules: Updates the worker Build ID assignment and redirect rules on the Task Queue.
+
+Provides various commands for adding, removing, or replacing worker Build ID assignment and redirect rules associated with a Task Queue. See the help of each sub-command for more.
+
+### temporal task-queue update-build-id-rules add-redirect-rule: Adds the rule to the list of redirect rules for this Task Queue.
+
+Adds a new redirect rule for this Task Queue. There can be at most one redirect rule for each distinct source build ID.
+
+#### Options
+
+* `--source-build-id` (string) - The source build ID for this redirect rule. Required.
+* `--target-build-id` (string) - The target build ID for this redirect rule. Required.
+* `--task-queue`, `-t` (string) - Name of the Task Queue. Required.
+* `--yes`, `-y` (bool) - Skip confirmation.
+
+### temporal task-queue update-build-id-rules commit-build-id: Completes the rollout of a Build ID for this Task Queue.
+
+Completes  the rollout of a BuildID and cleanup unnecessary rules possibly
+created during a gradual rollout. Specifically, this command will make the
+following changes atomically:
+	1. Adds an unconditional assignment rule for the target Build ID at the end of the list.
+	2. Removes all previously added assignment rules to the given target Build ID.
+	3. Removes any unconditional assignment rules for other Build IDs.
+
+To prevent committing invalid Build IDs, we reject the request if no pollers
+have been seen recently for this Build ID. Use the `force` option to disable this validation.
+
+
+#### Options
+
+* `--build-id` (string) - The target build ID to be committed. Required.
+* `--task-queue`, `-t` (string) - Name of the Task Queue. Required.
+* `--force` (bool) - Bypass the validation that pollers have been recently seen for this build ID.
+* `--yes`, `-y` (bool) - Skip confirmation.
+
+### temporal task-queue update-build-id-rules delete-assignment-rule: Deletes the rule at a given index in the list of assignment rules for this Task Queue.
+
+Deletes an assignment rule for this Task Queue. By default presence of one
+unconditional rule, i.e., no hint filter or percentage, is enforced, otherwise
+the delete operation will be rejected. Set `force` to true to bypass this
+validation.
+
+#### Options
+
+* `--task-queue`, `-t` (string) - Name of the Task Queue. Required.
+* `--rule-index`, `-i` (int) - Position of the assignment rule to be replaced. Required.
```

Comment:
Not sure this deserves a shortcut alias anymore than the rest of the required args on these calls that are fairly specific

Reply:
Yes, it is debatable, I just tried to add the shortcut for commonly used args across different (sub)commands